templates/definition/charger/lg-therma.yaml

```
@@ -0,0 +1,89 @@
+    en: |
+      Modbus connection to LG Therma V heat pump.
+      Uses SG Ready energy states for intelligent control.
+      Set the indoor unit in the installer settings > "Connectivity" > "Energy state" > "ESS use type" > to "Modbus".
+      Since the internal power measurement shows to high values, an external power measurement is suggested.
+      Factory setting Modbus (HEX) = 21 (33).
+      Modbus registers can be found within the installation manual (address without the leading number and -1).
+      Information on the energy state can be found within the installation manual. 
+      Energy states 6 and 7 can be adapted at the indoor unit in the installer settings > "Connectivity" > "Energy state" >"Energy state definition".
+params:
+  - name: modbus
+    choice: ["rs485"]
+    id: 33
+  - name: tempsource
+    required: true
+    type: choice
+    choice: ["water_tank", "water_inlet"]
+    default: water_tank
+    description: 
+      de: DHW Warmwassertank- oder Wassereinlasstemperatur
+      en: DHW warm water tank or inlet flow temperature   
+  - name: energystate
+    required: true
+    type: choice
+    choice: ["SGReady", "IndividualEnergystate"]
+    default: SGReady
+    description: 
+      de: SGReady oder anpassbare Energiezustände 6 und 7
+      en: SGReady states or modifyable energy state 6 and 7
+render: |
+  type: sgready
+  setmode:
+    source: map
+    values:
+      1: {{ if eq .energystate "SGReady" -}} 1 {{ else if eq .energystate "IndividualEnergystate" -}} 7 {{ else }} 1 {{- end }} # 1 Erzwungen Aus, 7 Energiesparmodus # Energiesparen
+      2: 2 # Normalbetrieb
+      3: {{ if eq .energystate "SGReady" -}} 3 {{ else if eq .energystate "IndividualEnergystate" -}} 6 {{ else }} 3 {{- end }} # 3 Ein-Empfehlung, 6 Ein-Empfehlung Schritt 1 # Boost
+    set:
+      source: modbus
+      {{- include "modbus" . | indent 4 }}
+      register:
+        address: 9
+        type: writeholdings
+        encoding: uint16
+  getmode:
+    source: map
+    values:
+      {{ if eq .energystate "SGReady" -}} 1 {{ else if eq .energystate "IndividualEnergystate" -}} 7 {{ else }} 1 {{- end }}: 1 # 1 Erzwungen Aus, 7 Energiesparmodus # Energiesparen
+      2: 2 # Normalbetrieb
+      {{ if eq .energystate "SGReady" -}} 3 {{ else if eq .energystate "IndividualEnergystate" -}} 6 {{ else }} 3 {{- end }}: 3 # 3 Ein-Empfehlung, 6 Ein-Empfehlung Schritt 1 # Boost
+    get:
+      source: modbus
+      {{- include "modbus" . | indent 4 }}
+      register:
+        address: 9
+        type: holding
+        encoding: uint16        
+  temp:
+    source: modbus
+    {{- include "modbus" . | indent 2 }}
+    scale: 0.1
+    register:
+      address:  {{ if eq .tempsource "water_tank" -}} 5 {{ else if eq .tempsource "water_inlet" -}} 2 {{ else }} 5 {{- end }} # 5 DHW Tank Temperatur, 2 Wassereinlasstemperatur
```

Remove unnecessary else if option since default is set
```suggestion
      address:  {{ if eq .tempsource "water_inlet" -}} 2 {{ else }} 5 {{- end }} # 5 DHW Tank Temperatur, 2 Wassereinlasstemperatur
```